code/script.js

```diff
@@ -0,0 +1,170 @@
+        <a href="https://www.facebook.com/anna.dimitrakopoulos" target="blank"><img src="${facebookIcon.src}" alt="${facebookIcon.alt}" class="facebook-icon"/></a>
+        `
+        })
+}
+
+//Function to fetch and display my repos.
+const fetchRepos = () => {
+    fetch(MY_REPOS)
+        .then((res) => res.json())
+        .then((data) => {
+            //Clear innerHTML to prevent duplicated repos.
+            projectsContainer.innerHTML = ``
+            const forkedRepos =
+                data.filter(repo => {
+                    return repo.fork && repo.name.startsWith("project-") && selectedLanguages.includes(repo.language)
+                })
+            //Function to sort repos by latest date.
+            forkedRepos.sort(function (a, b) {
+                return new Date(b.pushed_at) - new Date(a.pushed_at)
+            })
+            forkedRepos.forEach(repo => {
+                projectsContainer.innerHTML += `
+                      <div class="repo" id=${repo.name}>
+                        <p class="forked-repos"> <img src="${repoIcon.src}" alt="${repoIcon.alt}" class="repo-icon"/> ${repo.name} </p> 
+                        <a href="${repo.html_url}" target="blank">Go to repo!</a> 
+                        <p class="forked-repos"> <img src="${branchIcon.src}" alt="${branchIcon.alt}" class="branch-icon"/> ${repo.default_branch}</p>
+                        <p class="forked-repos"> <img src="${pushIcon.src}" alt="${pushIcon.alt}" class="push-icon"/> ${new Date(repo.pushed_at).toDateString()}</p>
+                        <p id="commit-${repo.name}">Amount of commits: </p> 
+                      </div>
+                        `
+            })
+
+            fetchPullRequests(forkedRepos)
+
+            //Only draw the chart once.
+            if (!chartDrawn) {
+                drawChart(forkedRepos.length)
+                chartDrawn = true
+            }
+        })
+}
+
+const fetchPullRequests = (repos) => {
+    repos.forEach(repo => {
+        const PULL_URL = `https://api.github.com/repos/Technigo/${repo.name}/pulls?per_page=100`
+
+        fetch(PULL_URL)
+            .then(res => res.json())
+            .then((data) => {
+                const myPullRequest =
+                    data.find(pull => pull.user.login === repo.owner.login)
+                if (myPullRequest) {
+                    fetchCommits(myPullRequest.commits_url, repo.name);
+                } else {
+                    document.getElementById(`commit-${repo.name}`).innerHTML =
+                        "No pull request yet!";
+                }
+            })
+    })
+}
+
+const fetchCommits = (myCommitsUrl, myRepoName) => {
+    fetch(myCommitsUrl)
+        .then(res => res.json())
+        .then(data => {
+            document.getElementById(`commit-${myRepoName}`).innerHTML += data.length
+        })
+}
+
+const filterLanguages = (event) => {
```

Comment:
This was a very nice feature and that you made it work! be proud! the code, even though it is a lot is clear and I sort of understands why it works, but I never would have enen tried to good job 💯